NuRadioReco/eventbrowser/dataprovider.py

```diff
@@ -1,4 +1,7 @@
-import NuRadioReco.eventbrowser.dataprovider_root
+try:
+    import NuRadioReco.eventbrowser.dataprovider_root
+except:
+    print("can't import RNO-G root reader")
```

Comment:
The dataprovider_root has not yet been updated for the new RNO-G reader, and we already removed the old one, leading to this error. This is already fixed on `rnog_eventbrowser`, I can make a PR next week. But it's probably okay to leave it like this.